client/src/components/HomeScreen.test.tsx

```diff
@@ -16,7 +16,7 @@ const apiCalls = {
     },
   },
   postNewAudit: (body: {}) => ({
-    url: '/api/election/new',
+    url: '/api/election',
```

Comment:
In this pattern, I'm used to a trailing slash for the URL to indicate a collection of elections that you're adding to.

Reply:
i know, but none of the other collections do that. eventually i want to change all of them to be plural